### PR TITLE
docs: add october showcase

### PR DESCRIPTION
**Pull request by 43081j**
Adds the october showcase blog post.

This is a **work in progress** since @jreinhold has some storybook wins to add to the post too

Also, @joyeecheung, would you mind reading to part about `require(esm)` and checking that I'm not talking nonsense? :joy:

@canadahonk, i've also mentioned porffor here. let me know if that's ok and the wording is right if you can 🙏

### CODE REVIEW

File: docs/blog/october-contributions-showcase.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+```ts
+// chai 5.x is ESM only
+const { expect } = require('chai')
+```
+
+Until now, many of us have been publishing "dual packages" - packages which contain both CommonJS and ES module syntax.
+
+This is unfortunate since it means we're often shipping double our package size in order to support both systems.
+
```
Comment (joyeecheung) [inline docs/blog/october-contributions-showcase.md:97]: By the way we are kicking off an initiative to better document the shipping practices and provide guides/troubleshooting for the migration:  https://github.com/nodejs/package-examples/issues/2 - kind of procrastinating it a bit until the collaboration summit session https://github.com/openjs-foundation/summit/issues/422, which would happen on Nov 8, 10:45 Dublin time, not sure remote participation is possible yet I hope it is! Though the session is mostly for awareness and brainstorming and I am sure we will get back to GitHub after it.

File: docs/blog/october-contributions-showcase.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+Great to see another large project taking a focus on performance :pray:
+
+They have already shaved off minutes from their build times :fire: very impressive work by the team.
+
+# Other news
+
+## `require(esm)` in Node
+
+[@joyeecheung](https://x.com/joyeecheung) and others have recently been working hard on allowing Node to `require(...)` ES modules.
+
```
Comment (joyeecheung) [inline docs/blog/october-contributions-showcase.md:86]: There is also the so-called faux-ESM package (authored in ESM, transpiled to CJS, likely only shipping the CJS version, and could be migrated to ship ESM directly). You can find a list of classified high-impact packages in https://github.com/wooorm/npm-esm-vs-cjs/ (there are some that are labeled incorrectly, e.g. some packages labeled as ESM are actually dual and still could use some cleanup https://github.com/wooorm/npm-esm-vs-cjs/issues/11 , but it's overall a very useful TODO list ;)

**Review by joyeecheung — APPROVED** [review]
Looks great, thanks for helping out and spreading the word! Some suggestions/additional info that may be interesting.

File: docs/blog/october-contributions-showcase.md (hunk 2 above)
Comment (43081j) [reply docs/blog/october-contributions-showcase.md:104]: aha good point. I totally forgot about the high impact repo too, that'll come in useful for tasking out the migrations

File: docs/blog/october-contributions-showcase.md (hunk 1 above)
Comment (43081j) [reply docs/blog/october-contributions-showcase.md:115]: would be super nice to join the summit if you can join remotely after all. I'd like to keep the community aligned with all this stuff, feels like we can do a lot here if we get it tasked out properly 🙏

**Comment by CanadaHonk** [comment]
Looks good, ty!

**Review by CanadaHonk — APPROVED** [review]

File: docs/blog/october-contributions-showcase.md [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+If you're using Node 16.x and above, you can drop this one too!
+
+## Docusaurus
+
+This one didn't originate from e18e but is still well worth a mention. [Docusaurus](https://github.com/facebook/docusaurus) is currently tackling a pretty huge project of [improving build performance](https://github.com/facebook/docusaurus/issues/10556).
+
+Great to see another large project taking a focus on performance :pray:
+
+They have already shaved off minutes from their build times :fire: very impressive work by the team.
```
Comment (bluwy) [inline docs/blog/october-contributions-showcase.md:116]:
```suggestion
They have already shaved off minutes from their build times :fire: very impressive work by [@sebastienlorber](https://x.com/sebastienlorber) and the team.
```

I think it was mostly Sebastien working on this, but I'll leave the "team here to be safe.

**Review by bluwy — COMMENTED** [review]
There's some weird spacing issues happening with the headings. I think avoiding using h1 (`#`) headings here, and instead start the main ones with h2 should fix it. The h1 is reserved for the blog title.

File: docs/blog/october-contributions-showcase.md [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+
+Instead of packaging a runtime alongside the code, porffor will compile down to native (which comes with huge performance gains).
+
+Recently, the biggest win has been to see the engine pass 50% of the Test262 tests! You can [read more about this here](https://goose.icu/porffor-50/).
+
+## e18e on Bluesky
+
+The e18e community now also has a profile on Bluesky: [e18e.dev](https://bsky.app/profile/e18e.dev).
+
+Give us a follow and we'll post updates on the cool stuff happening in the community! :butterfly:
```
Comment (patak-dev) [inline docs/blog/october-contributions-showcase.md:161]: 💙

**Comment by 43081j** [comment]
I changed it to use the lower headings. the previous posts used `h1`, though. so will just have to remember to avoid it in future

**Comment by bluwy** [comment]
I think we can update the previous blog posts too if we want to. The hash links should still be the same. Nonetheless the sidebar now looks much nicer with the categorization.

**Comment by 43081j** [comment]
you're right, I'll open a separate PR to update them if I get time soon 👍

File: docs/blog/october-contributions-showcase.md [hunk 5]
```diff
@@ -0,0 +1,165 @@
...
+
+In the web server space, [milliparsec](https://github.com/tinyhttp/milliparsec) has gained a huge amount of usage as a replacement for `body-parser`.
+
+At only 11kB, this is much smaller than older alternatives, and much faster!
+
+Excellent work by [v1rtl](https://github.com/talentlessguy) on building this and actively working on feedback from the community :heart:
+
+## Contributions / Improvements
+
+### Storybook
```
Comment (bluwy) [inline docs/blog/october-contributions-showcase.md:79]: Not sure if this is intentional as it doesn't have any content, but otherwise the rest LGTM

**Review by bluwy — APPROVED** [review]

File: docs/blog/october-contributions-showcase.md (hunk 5 above)
Comment (43081j) [reply docs/blog/october-contributions-showcase.md:79]: we're still waiting on the storybook team for a summary of a few changes so I can write this section FYI

**Comment by 43081j** [comment]
Good idea 👍 Will look a lot better on link previews

File: docs/blog/october-contributions-showcase.md [hunk 6]
```diff
@@ -0,0 +1,170 @@
...
+
+### ESLint plugins (and removing older node support!)
```
Comment (benmccann) [inline docs/blog/october-contributions-showcase.md:86]:
```suggestion
### ESLint plugins (and removing older Node support!)
```

File: docs/blog/october-contributions-showcase.md [hunk 7]
```diff
@@ -0,0 +1,165 @@
...
+In the web server space, [milliparsec](https://github.com/tinyhttp/milliparsec) has gained a huge amount of usage as a replacement for `body-parser`.
+
+At only 11kB, this is much smaller than older alternatives, and much faster!
+
+Excellent work by [v1rtl](https://github.com/talentlessguy) on building this and actively working on feedback from the community :heart:
+
+## Contributions / Improvements
+
+### Storybook
+
```
[the remark this answers is not in the document]
Comment (43081j) [reply docs/blog/october-contributions-showcase.md:85]: looks good to me. I reworded a tiny amount to fit with my wording but seems good

**Pull request merged**